File: ios/FindLocalDevices.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(FindLocalDevices, NSObject)

RCT_EXTERN_METHOD(getLocalDevices:(int)timeout
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/FindLocalDevices.swift
@objc(FindLocalDevices)
class FindLocalDevices: NSObject {

    @objc(getLocalDevices:withResolver:withRejecter:)
    func getLocalDevices(timeout: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
        resolve(a*b)
    }
}
